conf/logback-test.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+  ~ Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com>
+  -->
+<configuration>
+    <!-- Suppress logback complaining about multiple logback-test.xml files -->
+    <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
+                <pattern>%green(%date{ISO8601}) %gray(level=)[%highlight(%5level)] %gray(message=[%yellow(%message)])  %gray(logger=[%logger] thread=[%thread] rid=[%X{X-Request-ID}] user=[%X{Authorization}]) %replace(exception=[%xException]){'^exception=\[\]$',''}%n</pattern>
+            </encoder>
+        </encoder>
+    </appender>
+
+    <!-- switch on to debug tests -->
+    <root level="OFF">
```

Comment:
the scoverage statistics are affected by this 😢 it's saying our coverage isn't as high as it could be cos we not executing the log messages 🙄  we're the logs really too annoying to include?

Reply:
Not really, but test usually run fine without need for additional logging. I had to add it because I was not sure what was happening in the http calls for one of the tests so I added this to get some tracing which then helped me debug the test. We can switch it on if that improves coverage.

Reply:
switched on to improve improve test coverage